src/cdk-constructs/IxBucket.ts

```diff
@@ -12,34 +12,20 @@ export class IxBucket extends Bucket {
     id: ConstructId,
     props: ConstructProps = {},
   ) {
-    let bucketProps: ConstructProps = {}
+    const bucketProps: ConstructProps = {
+      blockPublicACLs: true,
+      cdk: {
+        bucket: {
+          enforceSSL: true,
+          ...props.cdk?.bucket,
+        },
+        ...props.cdk,
+      },
+      ...props,
+    }
     // Merge the provided props with the defaults.
     if (ixDeployConfig.isIxDeploy) {
-      bucketProps = {
-        ...props,
-        blockPublicACLs: true,
-        cdk: {
-          ...props.cdk,
-          bucket: {
-            ...props.cdk?.bucket,
-            enforceSSL: true,
-            encryption: BucketEncryption.S3_MANAGED,
-          },
-        },
-      };
-    } else {
-      bucketProps = {
-        ...props,
-        blockPublicACLs: true,
-        cdk: {
-          ...props.cdk,
-          bucket: {
-            ...props.cdk?.bucket,
-            enforceSSL: true,
-          },
-        },
-      };
+      bucketProps!.cdk!.bucket.encryption = BucketEncryption.S3_MANAGED
```

Comment:
Much clearer as to what changes here! Thanks. It might be good to also not encryption value if the user has explicitly set it themselfs, thus allowing them to override that default if they have a good reason to. Something like this:

```suggestion
      if (!('encryption' in bucketProps!.cdk!.bucket)) {
        bucketProps!.cdk!.bucket.encryption = BucketEncryption.S3_MANAGED
      }
```